src/api/app/models/bs_request.rb

```diff
@@ -730,7 +730,7 @@ def change_review_state(new_review_state, opts = {})
 
       # we want to check right now if pre-approved requests can be processed
       if go_new_state == :new && accept_at
-        Delayed::Job.enqueue AcceptRequestsJob.new
+        AcceptRequestsJob.perform_later
```

Comment:
Would be good to have a spec that makes sure this job gets called correctly here by the model.